src/graphql/schema.gql

```diff
@@ -210,6 +210,8 @@ input VoteWhere {
   vp_lte: Float
   vp_state: String
   vp_state_in: [String]
+  choice: Any
+  choice_in: [Any]
```

Comment:
I think the input choice can be a string, I am not sure if it will work with an object in case it is a weighted voting, but I will test it soon
```suggestion
  choice: String
  choice_in: [String]
```

Reply:
Hmm looks like doesn't work for weighted voting choices, Have you tried? should pass an object or a string?

Reply:
I tested locally only with strings

Reply:
Can you share some proposals with weighted voting choices?

Reply:
Example https://snapshot.org/#/aavegotchi.eth/proposal/0xade68c6cf0285a55e4cf914266761b9a505389ef4686e7972ad5426ac49cd238

Reply:
Thanks!
So we need filter like this
Types will be:  choice: string, choice_in: string[]
```
Object.keys(choice).filter(value => filter.choice_in.includes(value))
```

But we need to do it with sql [here](https://github.com/snapshot-labs/snapshot-hub/blob/master/src/graphql/helpers.ts#L101)

Reply:
I am bit confused but `buildWhereQuery` is used in many places we should make sure others are not affected 😅

Reply:
@bonustrack Any advice?

Reply:
Any chance we can move forward with this without @bonustrack?

Reply:
So as far as I remember Fabien was against the JSON search functions so want to confirm with him. Do you see any alternative ways @j0hnfl0w?

Reply:
Alternatives:
1. Search all votes client-side, seems impossible to preload 60k votes
2. Search by choice only for votes with simple choices (string/number)

Reply:
Maybe is it possible to convert column to string instead of JSON @bonustrack

Reply:
Even if we convert all these json to string how we are gonna search by json keys?

Reply:
Ahh yes, hiding my comment, then only JSON search should work. maybe you can continue with it. as Fabien not replying 😄